src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/ranges.hpp

```diff
@@ -44,6 +44,8 @@ static std::map<ov::NodeTypeInfo, std::vector<std::vector<InputGenerateData>>> i
         { ov::op::v4::Proposal::get_type_info_static(), {{{0, 1, 1000, 8234231}}, {{0, 1, 1000, 8234231}}} },
         { ov::op::v4::ReduceL1::get_type_info_static(), {{{0, 5}}, {{0, 5, 1000}}} },
         { ov::op::v4::ReduceL2::get_type_info_static(), {{{0, 5}}, {{0, 5, 1000}}} },
+        { ov::op::v1::Convolution::get_type_info_static(), {{{-1, 2}}, {{-1, 2, 1000}}} },
+        { ov::op::v1::DeformableConvolution::get_type_info_static(), {{{-1, 2}}, {{-1, 2, 1000}}} },
```

Comment:
First of all, if I correctly understand the call sequence, actual data are generated in: https://github.com/openvinotoolkit/openvino/blob/0c9abf43a91fbe2dc95974b6cb2651bd883d1b12/src/tests/ie_test_utils/common_test_utils/data_utils.hpp#L193-L207
That means that the `range` here stands for the amount of unique numbers generated by the random numbers generator. Thus, range [-1, 2] is resolved to a sequence {-1; 0; 1; 2}, which then is divided by the resolution value. Therefore, we have the final sequence {-0.001; 0; 0.001; 0.002}. This this wat we wont?
Yet another problem is the `bfloat16` numbers with reduced mantissa containing only 7 bits. If you take a look at the default flow, you will see that we use the range of [0, 2560] and the resolution of 256. This combination is resolved to the actual data range of [0, 10] with the step of 256. This configuration is important to avoid input data rounding when converting them to the `bfloat16` data type. With the step of 256, which is 2^8, we always have the mantissa no longer than 7 bits, so no rounding happens when converting to bf16.
Thus. we should decide whether the input tensor values in range [-1;2] are good enough to avoid fp16 values overflow. If so, let's just use the following parameters {-256, 512, 256}. If we wont the values to be significantly less than 1, it is better to have something like that {-256, 256, 1024} sot that the final range is [-0.25, 0.25], or increase the resolution number by a bigger power of two. The basic idea is to keep the mantissa no longer than 7 bits.

Reply:
@maxnick {-1, 2, 1000} would have -1 to 1 values with 0.002 resolution. It is -1, -0.998,......0.998, 1.  I'm not sure that we have bf16 test case for template plugin for the Convolution/DeformableConbolution. @iefode Do you knw it?

Reply:
Bf16 is supported only by CPU

Reply:
@maxnick Thank you for the comment! I updated range as you suggested. I didn't check bf16 case because there is no bf16 IR for Convolution/DeformableConvolution in Conformance IR.
BTW we need to interpret "range" with resolution to have actual result. {-1,2,256} is more easy to understand for {-1< n <1} generation.
pointer[i] = static_cast<T>(start_from + static_cast<T>(random.Generate(range * k)) / k);

Reply:
@iefode The range [-1,1] with resolution 256 in convolution causes over 2000 failed case in ConvolutionLayerCPUTest. Negative only[-1,0] or positive only[0,1] is ok. So I will use [0,1].

Reply:
@maxnick Some bf16 convolution layer tests fails again and again as I reduced range. Do you have any good suggestion for the tests? I don't have bf16 CPU machine(Xeon) on my local. So I pushes a change and check it on Azure CI.

Reply:
@maxnick Could you please help @wilson-seok with bf16 tests?